test/transaction_changes_test.rb

```diff
@@ -66,4 +66,27 @@ def test_transaction_changes_after_reload_unsaved
     assert_equal ["Dylan", "Dillon"], @user.stored_transaction_changes["name"]
   end
 
+  def test_transaction_changes_for_changing_updated_at
+    @user.update_attributes!(updated_at: Time.now - 1.second)
+    old_updated_at = @user.updated_at
+    @user.stored_transaction_changes = nil
+
+    @user.updated_at = Time.now
+    @user.save!
+
+    assert_equal [old_updated_at, @user.updated_at], @user.stored_transaction_changes["updated_at"]
+  end
+
+  def test_transaction_changes_for_touch
+    @user.update_attributes!(updated_at: Time.now - 1.second)
+    old_updated_at = @user.updated_at
+    @user.stored_transaction_changes = nil
+
+    @user.touch
+    if ActiveRecord::VERSION::MAJOR < 4
```

Comment:
I think we can just remove support for rails 3.2 now.  I think this gem is mostly used for identity_cache and it requires `activerecord >= 4.0.4`